### PR TITLE
complete KAS anchor feature (NEED TO REBASE THE LAST PR)

### PR DESCRIPTION
**Pull request by KimKyungup**
## Proposed changes

This PR is the last PR of KAS anchor feature.
Below works are included.
- Add anchor into subbridge
- Add `KASAnchor` API
- change chainHEAD event to chainEvent for Anchoring
- Add an exception of setting `KASAnchoirPeriod`

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [ ] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [ ] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by KimKyungup** [comment]
I will rebase this on the last PR.

File: console/web3ext/web3ext.go [hunk 1]
```diff
@@ -1012,6 +1012,11 @@ web3._extend({
 			call: 'subbridge_unsubscribeBridge',
 			params: 2
 		}),
+		new web3._extend.Method({
+			name: 'KASAnchor',
+			call: 'subbridge_kASAnchor',
```
Comment (ehnuje) [inline console/web3ext/web3ext.go:1017]:
```suggestion
			call: 'subbridge_kasAnchor',
```

File: cmd/utils/nodecmd/dumpconfigcmd.go [hunk 2]
```diff
@@ -287,6 +287,10 @@ func makeServiceChainConfig(ctx *cli.Context) (config sc.SCConfig) {
 	cfg.KASAnchor = ctx.GlobalBool(utils.KASServiceChainAnchorFlag.Name)
 	if cfg.KASAnchor {
 		cfg.KASAnchorPeriod = ctx.GlobalUint64(utils.KASServiceChainAnchorPeriodFlag.Name)
+		if cfg.KASAnchorPeriod == 0 {
+			cfg.KASAnchorPeriod = 1
+			logger.Warn("KAS anchor period is set by 1")
```
[the remark this answers is not in the document]
Comment (KimKyungup) [reply cmd/utils/nodecmd/dumpconfigcmd.go:292]: This case is that user gives `0` for this option. :)

File: console/web3ext/web3ext.go (hunk 1 above)
Comment (KimKyungup) [reply console/web3ext/web3ext.go:1017]: `subbridge_kASAnchor` looks werid. But this is for `subbridge.KASAnchor()`. :)

File: cmd/utils/nodecmd/dumpconfigcmd.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (ehnuje) [reply cmd/utils/nodecmd/dumpconfigcmd.go:292]: Then the message should be something like below to notify the user that the anchor period can't be set as 0.
"As KAS anchor period can't be set as 0, now KAS anchoring started with the period as 1"

**Comment by ehnuje** [comment]
> What's the difference between this and #591?

@KimKyungup Could you please check this comment?

**Comment by KimKyungup** [comment]
I will rebase this. Before that time, please take a look https://github.com/klaytn/klaytn/pull/597/commits/d83136292b43bfaac041a801a6818060090d681f.

**Pull request closed without merging**